### PR TITLE
[FEATURE] PHP 7.2 compatibility

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

**Comment by oliverklee** [comment]
This PR is blocked by https://github.com/phpList/rest-api/pull/36. Re-running the CI job after the merge should suffice.

**Comment by oliverklee** [comment]
Unblocked.

**Review by samtuke — APPROVED** [review]

**Pull request merged**
